scex-util/src/main/scala/com/avsystem/scex/util/function/FormatUtilImpl.java

```diff
@@ -22,25 +20,12 @@ public class FormatUtilImpl implements FormatUtil {
     private static final Pattern numericPattern = Pattern.compile(NUMERIC_PATTERN);
 
     public static String getPeriodMessage(Date from) {
```

Comment:
Seems unused, remove?

Reply:
There was a usage in UMP but it's easy enough to remove, sure.